python/lsst/resources/_resourcePath.py

```diff
@@ -409,6 +416,21 @@ def geturl(self) -> str:
         """
         return self._uri.geturl()
 
+    def to_fsspec(self) -> tuple[AbstractFileSystem, str]:
+        """Return an abstract file system and path that can be used by fsspec.
+
+        Returns
+        -------
+        fs : `fsspec.spec.AbstractFileSystem`
+            A file system object suitable for use with the returned path.
+        path : `str`
+            A path that can be opened by the file system object.
+        """
+        if fsspec is type:
```

Comment:
`fsspec` is `None` when it's not available, so this exception will never raise.  (This same issue is present in all the files.)

Reply:
Thanks -- that was from earlier version. The perils of not testing without it installed.